File: challenges/1-produto-simples/C/camilealheiro/ProdutoSimples.c
#include <stdio.h>

int main(void) {

  int num1, num2;
  printf("Digite dois inteiros: ");
  scanf("%d %d", &num1, &num2);
  
  int PROD = num1 * num2;
  printf("produto: %d", PROD);
  return 0;
}

File: challenges/1-produto-simples/php/tozeos/produtosimples.php
<?php

echo "Digite dois numerais separados por um espaço para realizar a operação." . PHP_EOL;

$input = explode(" ", fgets(STDIN));
$num1 = (int) $input[0];
$num2 = (int) $input[1];
$prod = $num1 * $num2;

echo "Produto: " . $prod . PHP_EOL;